src/main/scala/net/degoes/00-lambdas.scala

```diff
@@ -89,32 +91,32 @@ object Lambdas extends ZIOSpecDefault {
 
             val _ = (convertToString, countLength)
 
-            val numberOfDigits: Int => Int = ???
+            val numberOfDigits: Int => Int = countLength.compose(convertToString)
 
             assertTrue(numberOfDigits(123) == 3)
-          } @@ ignore +
+          } +
           /**
            * EXERCISE
            *
            * Use `identity` to create a function that returns its same string
            * argument.
            */
           test("identity") {
-            val sameString: String => String = ???
+            val sameString: String => String = a => a
```

Comment:
```suggestion
            val sameString: String => String = identity
```
There is a real function name `identity` that does the same thing